.github/workflows/append-release-cmake.yml

```diff
@@ -0,0 +1,76 @@
+name: Append Release CMake
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'Ref to be used as release'
+        default: 'latest'
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      ref:
+        description: 'Ref to be used as release'
+        default: 'latest'
+        required: true
+        type: string
+
+jobs:
+  generate_cmake_files:
```

Comment:
Maybe this should be part of the Release workflow.

Reply:
@dudoslav will you do this?

Reply:
Added to release workflow as a reusable workflow